spec/mspec/lib/mspec/matchers/include_any_of.rb

```diff
@@ -0,0 +1,31 @@
+class IncludeAnyOfMatcher
+  def initialize(*expected)
+    @expected = expected
+  end
+
+  def matches?(actual)
+    @actual = actual
+    @matched = []
+    @expected.each do |e|
+      if @actual.include?(e)
+        @matched.push e
+      end
+    end
+    return !@matched.empty?
+  end
+
+  def failure_message
+    ["Expected #{@actual.inspect}", "to include any of #{@expected.inspect}"]
+  end
+
+  def negative_failure_message
+    ["Expected #{@actual.inspect}", "not to include any of #{@matched.inspect}"]
+  end
+end
+
+# Cannot override #include_any_of at the toplevel in MRI
```

Comment:
This comment only applies for `main.include`, so it should be removed here.

Reply:
Fixing.